client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -19,7 +22,35 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 			'You enabled Jetpack and unlocked dozens of website-bolstering features. Continue preparing your site below.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		// TODO: adapt when we have more info
+		const siteRedirectHref = '#';
+
+		return (
+			<Fragment>
+				<DocumentHead title={ translate( 'Summary ‹ Jetpack Onboarding' ) } />
+				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+				<div className="steps__summary-columns">
+					<div className="steps__summary-column">
+						{ translate( "Steps you've completed:" ) }
+						<div className="steps__summary-column entry-completed">
+							<Gridicon icon="chevron-down" size={ 12 } />
+							{ translate( 'Site title and description' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Site Title and Description' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-onboarding-summary-include-content)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).